__tests__/app.test.js

```diff
@@ -40,4 +41,91 @@ describe('news-project', () => {
+                    expect(article).toHaveProperty('topic', expect.any(String));
+                    expect(article).toHaveProperty('created_at', expect.any(String));
+                    expect(article).toHaveProperty('votes', expect.any(Number));
+                    expect(article).toHaveProperty('article_img_url', expect.any(String));
+                })
+            })
+        })
+
+        test('GET: 200 - a get request should return array of objects with a new property "comment_count"', () => {
+            return request(app).get('/api/articles').expect(200)
+            .then(({body: {articles}}) => {
+                expect(articles.length).toBeGreaterThan(0);
+                articles.forEach(article => {
+                    expect(article).toHaveProperty('comment_count', expect.any(String));
+                })
+            })
+        })
+
+        test('GET: 200 - a get request should return array of objects sorted by "created_at" in a DESC order', () => {
+            return request(app).get('/api/articles/?sort_by=created_at').expect(200)
+            .then(({body: {articles}}) => {
+                expect(articles[0]).toHaveProperty('created_at', '2020-11-03T09:12:00.000Z')
+                expect(articles[articles.length - 1]).toHaveProperty('created_at', '2020-01-07T14:08:00.000Z')
+            })
+        })
+    })
+
+    describe('/api/articles/:article_id', () => {
+        test('GET: 200 - a get request should response with status 200', () => {
+            return request(app).get('/api/articles/2').expect(200);
+        })
+
+        test('GET: 200 - a get request should return an article object', () => {
+            return request(app).get('/api/articles/2').expect(200)
+            .then(({body: {article}}) => {
+                expect(article).toBeInstanceOf(Object);
+            })
+        })
+
+        test('GET: 200 - a get request should return an object with following properties', () => {
+            return request(app).get('/api/articles/5').expect(200)
+            .then(({body: {article}}) => {
+                expect(article).toEqual(expect.objectContaining({
+                    author : expect.any(String),
+                    title : expect.any(String),
+                    article_id : expect.any(Number),
+                    body : expect.any(String),
+                    topic : expect.any(String),
+                    created_at : expect.any(String),
+                    votes : expect.any(Number),
+                    article_img_url : expect.any(String)
+                }))
+               
+            })
+        })
+
+        test('GET: 404 - a get request should return a message "Not Found" when "article_id" does not exist', () => {
+            return request(app).get('/api/articles/100').expect(404)
+            .then(({body : {message}}) => {
+                expect(message).toEqual('Not Found')
+            })
+        })
```

Comment:
We also want to make sure to include a test for a 400 for when a ID is used that will never exist